lib/core/Axios.js

```diff
@@ -13,6 +14,7 @@ var mergeConfig = require('./mergeConfig');
  */
 function Axios(instanceConfig) {
   this.defaults = instanceConfig;
+  this.version = pkg.version;
```

Comment:
I'd rather putting it in just the global axios require instead of an instance, since the version is related to the import, let me know if this makes sense

Reply:
Strongly agree! Thanks 😄